public/app/features/explore/Table/TableContainer.tsx

```diff
@@ -68,53 +74,62 @@ export class TableContainer extends PureComponent<Props> {
       // differently and sidestep this getLinks API on a dataframe
       for (const frame of dataFrames) {
         for (const field of frame.fields) {
-          field.getLinks = (config: ValueLinkConfig) => {
-            return getFieldLinksForExplore({
-              field,
-              rowIndex: config.valueRowIndex!,
-              splitOpenFn,
-              range,
-              dataFrame: frame!,
-            });
-          };
+          if (field.type === FieldType.nestedFrames) {
```

Comment:
Hopefully, after https://github.com/grafana/grafana/pull/71811, this should not be needed any more.

Reply:
Hey @ifrost, I've merged with main but am seeing unexpected behaviour. Clicking on a subtable link refreshes explore and opens a new page and `traceIdHidden` but I don't see the variables being replaced.

<img width="696" alt="Screenshot 2023-08-01 at 09 09 48" src="https://github.com/grafana/grafana/assets/90795735/80f9b026-1b84-4fc7-af0d-95906f6fa53c">

<img width="972" alt="Screenshot 2023-08-01 at 09 00 41" src="https://github.com/grafana/grafana/assets/90795735/d32dd219-d70f-4e30-8656-ef854c553220">

Reply:
I can have a look. Are all changes pushed to the branch? What's the easiest way to set this up to see sub tables?

Reply:
All changes pushed now. You can run `make devenv sources=tempo` and run a query like in the following image to see the sub tables. Thanks!

![Screenshot 2023-08-01 at 09 59 55](https://github.com/grafana/grafana/assets/90795735/9c421a0e-32da-48f8-8c80-a74b5fdc2a17)

Reply:
Oh, it's because getLinksSupplier requires the same post-processor to be passed to make this work in context of Explore:

Apply changes in https://github.com/grafana/grafana/pull/71953/files#r1280456828 and it should work now!

```typescript
              valueField.getLinks = getLinksSupplier(
                nestedFrames[nfIndex],
                valueField,
                valueField.state!.scopedVars,
                context.replaceVariables,
                options.timeZone,
                options.dataLinkPostProcessor
              );
```

Reply:
Ahh of course! That did it, thanks Piotr.